src/java.base/share/classes/jdk/internal/foreign/abi/aarch64/CallArranger.java

```diff
@@ -32,6 +32,7 @@
 
 package jdk.internal.foreign.abi.aarch64;
 
+import java.lang.foreign.AddressLayout;
```

Comment:
Please remove
```java
import sun.security.action.GetPropertyAction;
```
and
```java
    private static final boolean isWinOS = GetPropertyAction.privilegedGetProperty("os.name").startsWith("Windows");
```
replacing `isWinOS` with `Utils.IS_WINDOWS`.

Reply:
Fixed as suggested above.

Reply:
The (unused) import is still present.

Reply:
removed `import sun.security.action.GetPropertyAction;`.